rfcs/0001-meta-schedule-autotensorir.md

```diff
@@ -147,17 +203,20 @@ sch.reorder(
 
 ### 3.4. AutoScheduler-style Design Space Generation
 
-AutoScheduler (Ansor) generates schedule templates by applying their SearchRules to each stage.
-SearchRule analyzes TE and eagerly trigger schedule primitives accordingly in its internally
-maintained mini IR.
+To generate design space, AutoScheduler (Ansor) applies a set of rules to each TE stage.
+The rules analyze the TE operations and apply an internal DSL to manipulating its internal IR,
+which is in the end mapped to TE schedule primitives. This process is called *sketch generation*.
 
-As introduced in Section 3.2, composite schedule rules are equivalent to AutoScheduler's SearchRule
-in TensorIR scheduling. To further generate a design space for scheduling, sampling instructions are
-used in composite schedule rules. Similarly, the sketch generation phase in AutoScheduler is
-equivalent to applying composite schedule rules to each block in TensorIR.
+Composite schedule rules work in a similar way scheduling TensorIR, as introduced in Section 3.2.
+It analyzes the TensorIR and apply schedule primitives directly to TensorIR accordingly.
+When applying such rules to each TensorIR block in certain order (e.g. Post-DFS Order),
+it generates a sequence of schedule primitives.
+If the sampling instructions are present in this sequence, 
+the support of the probability space form a design space of possible schedulings.
```

Comment:
Further exploring the space is not part of the sketch generation phase (1st phase in AutoScheduler), instead it corresponds to the random annotation phase (2nd phase in AutoScheduler). I rephrased it a bit so that it is more clarified.